test/nodetool/test_compact.py

```diff
@@ -76,3 +77,22 @@ def test_user_defined(nodetool, scylla_only):
              "me-3g8w_11cg_4317k2ppfb6d5vgp0w-big-Data.db"),
             {},
             ["error processing arguments: --user-defined flag is unsupported"])
+
+
+@pytest.mark.parametrize("flush", ("true", "false"))
+def test_keyspace_flush(nodetool, scylla_only, flush):
```

Comment:
It's not clear what a "keyspace flush" actually test? It doesn't test "nodetool flush"... Please add a comment explaining that it actually tests a specific parameter of "nodetool compact". Maybe also use "compact" in the test's name.
And please also explain why it's a Scylla-only test, because it tests a nodetool option "--flush-memtables" that didn't exist in Cassandra's nodetool.

Reply:
> It's not clear what a "keyspace flush" actually test? It doesn't test "nodetool flush"... Please add a comment explaining that it actually tests a specific parameter of "nodetool compact".

I'll rename the function to mention `flush_memtables_option`.

> Maybe also use "compact" in the test's name.

test_compaction.py is all about `scylla nodetool compact` and it's not mentioned in any other test case.
I think that given the mention of the specific option in the function name, this isn't requried.

> And please also explain why it's a Scylla-only test, because it tests a nodetool option "--flush-memtables" that didn't exist in Cassandra's nodetool.

No problem.

Reply:
done in v7